scaleway/resource_k8s_cluster_beta.go

```diff
@@ -0,0 +1,733 @@
+package scaleway
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1beta3"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+
+	"gopkg.in/yaml.v2"
+)
+
+func resourceScalewayK8SClusterBeta() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceScalewayK8SClusterBetaCreate,
+		Read:   resourceScalewayK8SClusterBetaRead,
+		Update: resourceScalewayK8SClusterBetaUpdate,
+		Delete: resourceScalewayK8SClusterBetaDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		SchemaVersion: 0,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: "The name of the cluster",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "The description of the cluster",
+			},
+			"version": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Optional:    true,
+				Default:     nil,
+				Description: "The version of the cluster",
+			},
+			"cni": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The CNI plugin of the cluster",
+			},
+			"enable_dashboard": {
+				Type:        schema.TypeBool,
+				ForceNew:    true,
+				Optional:    true,
+				Computed:    true,
+				Default:     nil,
+				Description: "Enable the dashboard on the cluster",
+			},
+			"ingress": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+				Default:     nil,
+				Description: "The ingress to be deployed on the cluster",
+			},
+			"tags": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional:    true,
+				Description: "The tags associated with the cluster",
+			},
+			"autoscaler_config": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Computed:    true,
+				Description: "The autoscaler configuration for the cluster",
+				Elem:        autoscalerConfigSchema(),
+			},
+			"default_pool": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Computed:    true,
+				Optional:    true,
+				Description: "Default pool created on the cluster on creation",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"node_type": {
+							Type:        schema.TypeString,
+							Required:    true,
+							ForceNew:    true,
+							Description: "Server Type of the default pool servers",
+						},
+						"autoscaling": {
+							Type:        schema.TypeBool,
+							Optional:    true,
+							Default:     nil,
```

Comment:
Set the default value here (true or false), and remove the Computed.  (might not work in a list though)